Duplicati/CommandLine/Commands.cs

```diff
@@ -234,14 +236,84 @@ public static int Affected(TextWriter outwriter, Action<Duplicati.Library.Main.C
             }
         }
 
-        public static List<string> PrefixArgsWithAsterisk(IList<string> argList)
+        /// <summary>
+        /// For bare file names with no wildcards, replace argument with
+        /// the equivalent of prefixing with "*/" so we search all folders.
+        /// </summary>
+        public static IEnumerable<string> PrefixArgsWithAsterisk(IEnumerable<string> argList) =>
+            argList.Select(PrefixArgWithAsterisk);

```

Comment:
This is simple enough to be on one line.

Reply:
Fixed in latest commit.